### PR TITLE
Codeship CI/CD

### PR DESCRIPTION
**Pull request by FatherShawn**
This partially tests in the Jet local tool.  The composer install command works in the container, but not in the jet simulation.  We should check this over and then give it a try in codeship.

### CODE REVIEW

**Comment by jdleonard** [comment]
@FatherShawn I merged this through to the master branch so that it will run on codeship (codeship tag set to master). It seems like composer install is hanging.

File: codeship-steps.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        - service: front-end
+          name: npm_install
+          command: npm ci
+        - service: front-end
+          name: build_theme
+          command: npm run build
+- name: deploy
+  tag: master
+  service: deploy
+  command: vendor/bin/robo deploy --remote-branch=newbrunswick
```
Comment (jdleonard) [inline codeship-steps.yml:21]: @FatherShawn I also wasn't certain about "newbrunswick" here

Comment (FatherShawn) [reply codeship-steps.yml:21]: LOL - paste error That should come out and it will default to master

**Comment by FatherShawn** [comment]
Argh - well I've never tried the parallel approach before.  I use the composer command all the time so I'll try the simpler sequential definition and see if that works.  Shelling into the container and running it works fine.

**Comment by jdleonard** [comment]
Composer worked for me when I was playing around at https://github.com/Drupal-NYC/drupalcampnyc/blob/drupal-europe-ci/codeship-steps.yml

**Comment by FatherShawn** [comment]
cool - I'll compare

**Pull request merged**